test/lib/omniauth/strategies/openid_connect_test.rb

```diff
@@ -45,7 +45,7 @@ def test_logout_phase_with_discovery
       end
 
       def test_logout_phase_with_discovery_and_post_logout_redirect_uri
-        expected_redirect = 'https://example.com/logout?post_logout_redirect_uri=https%3A%2F%2Fmysite.com'
+        expected_redirect = 'https://example.com/logout?post_logout_redirect_uri=https%3A%2F%2Fmysite.com&id_token_hint'
```

Comment:
Should this value be blank? Should the key be omitted entirely if there is no token?

Reply:
I'm not sure whether it's worth the extra effort to create a test case for the second case.